README.md

```diff
@@ -65,6 +66,10 @@ sudo apt-get install battery-monitor -y
 ```
 That's all. **Battery Monitor (Stable)** is installed on your system.
 
+### For Arch Linux and it's derivatives
```

Comment:
It will be great if you mention the commands with which our users can install Battery Monitor in Arch easily. You can follow the current one for Ubuntu.